retriever/try_install_modified.py

```diff
@@ -0,0 +1,139 @@
+"""Attempt to update datasets in all database management systems.
+This module, when run, attempts to install datasets from modified Retriever
+scripts in the /scripts folder (except for those listed in IGNORE), for each
+engine in ENGINE_LIST() from __init__.py. In other words, it trys to install
+using all possible combinations of database platform and script and checks
+to see if there are any errors. It does not check the values in the database.
+"""
+from __future__ import print_function
+from __future__ import absolute_import
+import os
+import sys
+from imp import reload
+from retriever.lib.tools import choose_engine
+from retriever import MODULE_LIST, ENGINE_LIST, SCRIPT_LIST
+from retriever.version import get_module_version, update_version_file
```

Comment:
```version.py``` is in the parent directory, so directly importing will not work.
Using ```os```, ```sys``` ```version``` could be imported changing the relative path. But I doubt it is a good way to implement. Can functions ```get_module_version```, ```update_version_file``` be added in ```retriever``` somehow? Because a function like ```update_version_file``` could be very handy when implementing a python interface.